src/components/scenes/PluginViewScene.js

```diff
@@ -304,14 +379,15 @@ class PluginView extends React.Component<PluginProps, PluginState> {
           if (this.openingSendConfirmation) {
             return
           }
+
           this.openingSendConfirmation = true
+          // console.log('a1: data ', data)
```

Comment:
Extraneous debug logging. Please remove for production

Reply:
fixed